CMakeLists.txt

```diff
@@ -214,7 +214,7 @@ install(FILES "scap-workbench.appdata.xml"
 if (ASCIIDOCTOR_EXECUTABLE)
     file(GLOB USER_MANUAL_SCREENSHOTS "${CMAKE_CURRENT_SOURCE_DIR}/doc/user_manual/*.png")
     add_custom_command(
-        OUTPUT doc/user_manual.html
+        OUTPUT ${CMAKE_CURRENT_SOURCE_DIR}/doc/user_manual.html
```

Comment:
There is a very good reason why we  do that. It's so that when you use the tarball you don't have to have `asciidoctor` installed. `asciidoctor` is awesome but not packaged on all our target distributions, including RHEL.